safe_landing_planner/include/safe_landing_planner/waypoint_generator.hpp

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include <avoidance/usm.h>
+#include <safe_landing_planner/grid.hpp>
+
+#include <Eigen/Dense>
+
+#include <functional>
+#include <vector>
+
+namespace avoidance {
+
+const std::vector<Eigen::Vector2f> exploration_pattern = {
+    Eigen::Vector2f(1.f, 0.f),  Eigen::Vector2f(1.f, 1.f),
+    Eigen::Vector2f(0.f, 1.f),  Eigen::Vector2f(-1.f, 1.f),
+    Eigen::Vector2f(-1.f, 0.f), Eigen::Vector2f(-1.f, -1.f),
+    Eigen::Vector2f(0.f, -1.f), Eigen::Vector2f(1.f, -1.f)};
+
+enum class SLPState {
+  goTo,
```

Comment:
In the example test, the convention for states is UPPER_CASE. Can we be consistent?